### ISSUE TITLE
Passwort-Rücksendprozess auf Login-Seite erklären

### ISSUE THREAD

**Issue by oliveraustria** [Ferienhort/F.H.I.E.L.D.#18]

**Comment by GLSteiner** [comment]
etwa so?

![olivernerven](https://cloud.githubusercontent.com/assets/23318440/26271085/aedc1fb0-3d01-11e7-9ae8-16b25079acc5.png)

**Comment by GLSteiner** [comment]
Grundsatzlich eine Gute Idee, aber ich denke fur ein Ferienhort internes Tool nicht notig - es sollte pro Bereich klar sein, wer Ansprechperson ist

**Comment by oliveraustria** [comment]
@GLSteiner Ja, ich denke das wäre schon perfekt - sonst gibt es ein Chaos bei jeder neuen Aktion. Also ein "Frag Uli", vielleicht sogar noch mit Telefonnummer und/oder Mailadresse. DANKE!

**Comment by GLSteiner** [comment]
Ich denke gerade in die andere Richtung...

Es ist kein geringer Aufwand zu organisieren wer - was - wann dokumentiert (mit oder ohne FHIELD, das gilt genau so für excel listen in der Dropbox!). Zu schauen, dass das eigene Team Zugangsdaten hat fällt dabei wirklich nicht ins Gewicht, zumal sich die Betreuer ja gegenseitig zurücksetzen können. 

Es ist nicht mit Uli ausgemacht, dass sie genau weiß, wer in welchem Bereich was zu erledigen hat. Und das ist eigentlich eine riesige Fehlerquelle, wo ein schlecht organisierterer Bereich dann andere runterreissen kann - "da kann ja dann quasi jeder daherkommen", denn ein "Frag Uli" hilft ja auschließlich:

- Schülern
- Betreuern, die nicht mal genug in einen Bereich eingearbeitet sind um zu wissen wer sonst noch in dem Bereich engagiert ist. 

Alle andere kennen ja hoffentlich ihre Bereichsleiter, Ansprechpersonen und Kollegen vor Ort. 

(und ja, ich weiß, der "Frag Uli" Vorschlag kam von mir, aber beim längeren nachdenken denke ich es  ist eine schlechte Idee, dass auf sie abzuwälzen)

Ich bin mir auch bewusst, das es nicht ersichtlich und nicht 100% intuitiv ist sich im Falle eines vergessenen PINs an einen Kollegen oder Bereichsleiter zu wenden.

Vorschläge:
- Ein Nachricht "PIN vergessen? Frage deine/n Bereichsleiter*Inn"
- Unter Umständen mit einer Liste der Bereichsleiter, wenn erwünscht.
- Eine Funktion, mit der PINs per Email zugeschickt werden ABER: irgendjemand muss die EMail Adressen dann Einpflegen! 
- Oder einfach die Bereichsleiter dafür verantwortlich machen, ihren Leuten zu erklären, dass sie sich gegenseitig zurücksetzen können.....

**Comment by oliveraustria** [comment]
Also ich finde "Ein Nachricht "PIN vergessen? Frage deine/n Bereichsleiter*Inn" perfekt. Aus meiner Sicht sollten wir immer MVP denken: https://de.wikipedia.org/wiki/Minimum_Viable_Product und auf der anderen Seite keine zu großen Fragezeichen bei den BenutzerInnen hinterlassen.
DANKE!

**Comment by GLSteiner** [comment]
Zuerst einmal: Done - ich habe aber nach PRAK, nicht nach kuume gegendert.

Ja, bin da sehr bei dir. Ich würde hier das hier allerdings gerne Sicherheit (Datenintegrität) als 3ten Punkt hinzufügen. Gerade "Wie bekomme ich eigentlich einen PIN" grenzt schon fast an https://de.wikipedia.org/wiki/Security_through_obscurity
